packages/CodeGenLib/src/Database/manage-metadata.ts

```diff
@@ -25,6 +31,93 @@ export class ManageMetadataBase {
       return this._newEntityList;
    }
 
+   public async enableMetaDataLogging(config: MetadataLoggingConfig): Promise<void> {
```

Comment:
@JS-BC  a better function name might be something like `handleMetadataLogging` - as you have named it it "sounds like" this either enables or disables logging, when it just does the logging (or not) based on the configuration.

Reply:
actaully maybe I don't understand what this function is doing